src/main/java/com/github/jreddit/action/ProfileActions.java

```diff
@@ -140,6 +141,9 @@ public UserInfo getUserInformation() throws ActionFailedException {
     public UserInfo about(String username) throws ActionFailedException {
 
         // Send GET request to get the account overview
+      
+        if(username == "[deleted]")
```

Comment:
can you add curly brackets to the if statement? it good practice so you don't end up with weird bugs in the future